src/test_l1_dsVideoDevice.c

```diff
@@ -82,15 +82,15 @@ static int gTestID = 1;
     if(value != comparison){\
         UT_LOG("\n In %s Comparison: [%d = %d]\n", __FUNCTION__, value, comparison);\
         dsVideoDeviceTerm();\
-        UT_FAIL();\
+        UT_FAIL_FATAL();\
     }\
 }\
 
 #define DS_ASSERT_AUTO_TERM_STRING(value, comparison){\
     if(strcmp(value, comparison) != 0){\
         UT_LOG("\n In %s Comparison: [%s = %s]\n", __FUNCTION__, value, comparison);\
         dsVideoDeviceTerm();\
-        UT_FAIL();\
+        UT_FAIL_FATAL();\
```

Comment:
please remove all the FATAL's as part of the upgrade.

You should fall through every test, not change to EQUAL_FATAL.